lms/templates/credit_notifications/credit_eligibility_email.html

```diff
@@ -61,8 +65,10 @@
         <tr>
           <td class="cn-footer-content">
             <p>
-              ${_(u"For more information on credit at edX, click {link}.").format(
-                link=u'<a href="https://www.edx.org/gfa">here</a>'
+              ${_(u"For more information on credit at {platform_name}, click {link_start}here{link_end}.").format(
```

Comment:
Can we change this to `{link_start}Click here for more information on credit at {platform_name}.{link_end}`? Again, it's more accessible to link the whole phrase.